### DISCUSSION TITLE
Open links in new window

### DISCUSSION THREAD

**Discussion by 1nfinite9** [shuding/nextra#1181] [Q&A] [answered]
Is there any way to provide an option to open just a regular Markdown link in a new window? I understand this is currently available for links in the menu and sidebar, but would be quite useful when using the docs theme to create a collection of links, both internal and external.

**Comment by shuding** [comment, marked as answer]
Currently if you use absolute links such as `https://example.com` it will open in new window. Generally the Markdown syntax is limited so you can't add extra attributes to links, and most people need in-place link for internal relative URLs.

**Reply by 1nfinite9** [reply]
Ah! I see why it's not working now. Some of the internal links I'm trying to use are http instead of https. Is there any workaround for this?

**Reply by 1nfinite9** [reply]
@shuding Any updates on this?

**Reply by verekia** [reply]
@1nfinite9 @shuding PR created: https://github.com/shuding/nextra/pull/1533
